RFCs/0006-DataCollection-Protocol.md

```diff
@@ -0,0 +1,56 @@
+# 006 DataCollection Protocol
```

Comment:
While this describes nicely the entire architecture, do we plan to have one more for folks writing an adapter? on the interfaces etc.